### PR TITLE
Feature/april 11 updates

### PR DESCRIPTION
**Pull request by Sarveshuk**
#### 4 commits are part of this PR. 

1. Navbar updates
Changed the way the navbar looks. Giving it a nice dark gray look. Fixed it's functionality. 
2. Company info updates
Changed the way the first section looks. The background image is now new and looks better. Added some company info here. 
3. Services section updates
No more lorem ipsum. Actual services with description is now available.
4. Contact us updates
Contact section is now different. It look different and hopefully works. I incorporated .php code for the mail function. Sadly, can't test that on my local machine, it has to be uploaded on the server. Will test functionality once it's merged and come back to it if it doesn't work. 

#### What else is needed?

1. Javascript 
2. More images
3. More details
4. Steps (Step 1: Contact us >> Step 2: Sit with us and let us analyze your business >> Step 3: Grow your business)

### CODE REVIEW

**Comment by prathapshetty-official** [comment]
Test

**Review by prathapshetty-official — COMMENTED** [review]
Test 2

File: index.html [hunk 1]
```diff
@@ -23,75 +24,78 @@
...
+
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item">
+                        <a class="nav-link" href="#features">Services</a>
+                    </li>
+                    <!-- <li>
+                        <a class="nav-link" href="#clients">Clients</a>
+                    </li> -->
+                    <li class="nav-item">
+                        <a class="nav-link" href="#contact-us">Contact us</a>
```
[the remark this answers is not in the document]
Comment (Bagurp) [reply index.html:48]: @Sarveshuk

File: index.html [hunk 2]
```diff
@@ -2,10 +2,11 @@
 <html>
 <head>
     <meta charset='utf-8'>
-    <title>PaddleBiz</title>
+    <title>PaddleBiz Business Solutions</title>
     <meta name='viewport' content='width=device-width, initial-scale=1'>
     <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
-    <link href="https://fonts.googleapis.com/css?family=Merriweather|Montserrat|Sacramento&display=swap" rel="stylesheet">
```
Comment (prathapshetty-official) [inline index.html:8]: @Sarveshuk line 8 test

**Review by prathapshetty-official — APPROVED** [review]
Looks good to me

File: index.html [hunk 3]
```diff
@@ -119,12 +123,22 @@ <h2>Clients</h2>
...
+            <form action="mail.php" method="post">
+                <label for="fname">First Name</label>
+                <input type="text" id="fname" name="firstname" placeholder="Your name..">
+                <label for="lname">Last Name</label>
+                <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+                <label for="email">Email</label>
+                <input type="text" id="email" name="email" placeholder="Your email address.">
+                <label for="subject">Subject</label>
+                <textarea id="subject" name="subject" placeholder="Write something.." style="height:170px"></textarea>
+                <input type="submit" value="Submit">
```
[the remark this answers is not in the document]
Comment (Sarveshuk) [reply index.html:140]: Good thought. I don't know how to do this with my limited understanding of php. We will have to create a new ticket for this and look into this deeply.

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sarveshuk) [reply index.html:48]: This should be fixed in my latest commit

File: index.html [hunk 4]
```diff
@@ -23,75 +24,78 @@
...
+            </div>
+        </nav>
     </div>
 
     <!-- Company info -->
     <div class="content">
 
         <div class="image-container">
-            <img id="main-image" class="main-img" src="static/images/main-image-1.jpg" alt="Image of a paddle" />
+            <img id="main-image" src="static/images/main-image-3.jpg" alt="Image of a paddle" />
```
[the remark this answers is not in the document]
Comment (Sarveshuk) [reply index.html]: This should be fixed in my latest commit

File: index.html [hunk 5]
```diff
@@ -23,75 +24,78 @@
...
             <div class="image-text">
-                <p>PaddleBiz Business Solutions</p>
-                <h1>Please fill this up.</h1>
+                <p>We help customers</p>
+                <h1>with radical and revolutionary solutions</h1>
```
[the remark this answers is not in the document]
Comment (Sarveshuk) [reply index.html]: This should be fixed in my latest commit

File: index.html [hunk 6]
```diff
@@ -23,75 +24,78 @@
...
 
                 <div class="feature-box col-lg-4">
                     <img class="service-img" src="static/images/service_3.png"/>
-                    <h3 class="feature-title">Investing</h3>
-                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit,
-                            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
-                            quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                    <h3 class="feature-title">Investing and staffing</h3>
+                    <p>Learn how and where to get an influx of money to grow your business.
+                        Need a staffing resource but have no money or time to hire this resource? We can help you.</p>
```
[the remark this answers is not in the document]
Comment (Sarveshuk) [reply index.html]: This should be fixed in my latest commit

File: main.css [hunk 7]
```diff
@@ -160,6 +156,64 @@ h2 {
...
+@media screen and (max-width: 600px) {
+  .column, input[type=submit] {
+    width: 100%;
+    margin-top: 0;
+  }
+}
+
 /* ------------------End contact us section------------------- */
 
 /* -------------------Start mobile section------------------- */
```
[the remark this answers is not in the document]
Comment (Sarveshuk) [reply main.css:219]: I have to fix this. Will take more time. I'll capture this on a ticket

File: main.css [hunk 8]
```diff
@@ -160,6 +156,64 @@ h2 {
...
+
+/* Clear floats after the columns */
+.row:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+/* Responsive layout - when the screen is less than 600px wide, make the two columns stack on top of each other instead of next to each other */
+@media screen and (max-width: 600px) {
```
[the remark this answers is not in the document]
Comment (Sarveshuk) [reply main.css:210]: This should be fixed in my latest commit

**Review by Bagurp — APPROVED** [review]

**Pull request merged**